src/snowflake/snowpark/dataframe_writer.py

```diff
@@ -37,6 +39,25 @@
 else:
     from collections.abc import Iterable
 
+option_aliases = {
+    "DELIMITER": ("FIELD_DELIMITER", lambda val: val),
+    "SEP": ("FIELD_DELIMITER", lambda val: val),
+    "LINESEP": ("RECORD_DELIMITER", lambda val: val),
+    "PATHGLOBFILTER": ("PATTERN", lambda val: val),
+    "QUOTE": ("FIELD_OPTIONALLY_ENCLOSED_BY", lambda val: val),
+    "NULLVALUE": ("NULL_IF", lambda val: val),
+    "DATEFORMAT": ("DATE_FORMAT", lambda val: val),
+    "TIMESTAMPFORMAT": ("TIMESTAMP_FORMAT", lambda val: val),
+}
+
+copy_options = [
+    "OVERWRITE",
+    "SINGLE",
+    "MAX_FILE_SIZE",
+    "INCLUDE_QUERY_ID",
+    "DETAILED_OUTPUT",
+]
```

Comment:
How is this different from `COPY_OPTIONS` in `snowflake.snowpark._internal.utils`? Would you mind adding it to the utils file instead?

Reply:
Sure no problem! Good suggestion.